pkg/controller/lifecycle/actuator.go

```diff
@@ -592,13 +593,13 @@ func getSeedResources(lakomReplicas *int32, namespace, genericKubeconfigName, sh
 	return resources, nil
 }
 
-func getShootResources(webhookCaBundle []byte, namespace, shootAccessServiceAccountName string) (map[string][]byte, error) {
+func getShootResources(webhookCaBundle []byte, extensionNamespace, shootAccessServiceAccountName, shootNamespace string) (map[string][]byte, error) {
```

Comment:
The namespace `shoot--<project>--<shoot>` is called shoot or seed namespace, especially in the context of the seed cluster (where usually the extensions are working). To avoid some confusion, let's rename the variable `shootNamespace` to `projectNamespace`.

```suggestion
func getShootResources(webhookCaBundle []byte, extensionNamespace, shootAccessServiceAccountName, projectNamespace string) (map[string][]byte, error) {
```

Reply:
Ah, thanks! I guess I'm confusing:
- shoot namespace (meaning the namespace in which resources for this specific shoot are contained)
- project namespace (namespace in which the "shoot" resource is contained)